### PR TITLE
removing the health analyzer from medics' pda

### PR DESCRIPTION
**Pull request by Verslebas**
## About the PR

It's not logical that the pda has a health analyzer, it devalues the regular analyzer because the analyzer in the pda even doesn't require a battery.

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Verslebas
- remove: Removed health analyzer from medics' pda 
- tweak: Changed the battery in the health analyzer from a small to a medium one

### CODE REVIEW

**Comment by LankLTE** [comment]
1984
In all seriousness I would advise either making this a cartridge or putting a medium battery in the health analyzer. At the moment they run out of power quite fast, and do serve a purpose in being usable by anyone who isn’t a doctor. If medbay is relying on them they shouldn’t be running out of power 24/7 esp. since not all medbays have battery chargers

**Comment by Verslebas** [comment]
> 1984
> In all seriousness I would advise either making this a cartridge or putting a medium battery in the health analyzer. At the moment they run out of power quite fast, and do serve a purpose in being usable by anyone who isn’t a doctor. If medbay is relying on them they shouldn’t be running out of power 24/7 esp. since not all medbays have battery chargers

I can replace the battery, it is a good idea, I also don't like that analyzers run out of battery quickly, but a medical scanner in pda is weird

**Comment by EmoGarbage404** [comment]
> If medbay is relying on them they shouldn’t be running out of power 24/7 esp. since not all medbays have battery chargers

sounds like a reason for them to get better batteries and for mappers to map in chargers.
just because something is the status quo doesn't mean it should be maintained.

**Comment by Aisu9** [comment]
> > If medbay is relying on them they shouldn’t be running out of power 24/7 esp. since not all medbays have battery chargers
> 
> sounds like a reason for them to get better batteries and for mappers to map in chargers. just because something is the status quo doesn't mean it should be maintained.

Because mappers had no reason to map charger? what about the defibrillators that were map without chargers in medbay?

**Comment by LankLTE** [comment]
> sounds like a reason for them to get better batteries and for mappers to map in chargers. just because something is the status quo doesn't mean it should be maintained.

You can literally find medium capacity batteries in the super readily-available flashlights spread across the station. I’m only suggesting health scanners start with one because it’s simple to replace but sort of screws over people who don’t know they can / should. There has also been a reason (defibs) for chargers to be mapped for months and they still aren’t on a few stations

**Comment by Verslebas** [comment]
write it like that?

> If this does go forward, please revise your Changelog entry according to the Changelog guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog

**Comment by Emisse** [comment]
can you change health analyzer back to small battery, incentivizes people actually going and getting better batteries + most maps have medbays with rechargers

**Comment by chromiumboy** [comment]
I think this change should be made, but only once we have a stronger system in place for diagnosing patients via other means

**Comment by Verslebas** [comment]
> can you change health analyzer back to small battery, incentivizes people actually going and getting better batteries + most maps have medbays with rechargers

Sure

**Comment by JoeHammad1844** [comment]
> can you change health analyzer back to small battery, incentivizes people actually going and getting better batteries + most maps have medbays with rechargers

imagine getting shot and there are no medics in medbay because theyre all out getting batteries.

this will also skyrocket the demand for flashlights and science can't print medium capacity batteries without gold (but to be fair they can print ten with one bar)

even if they do they're probably all going to go towards borgs and not the crew

File: Resources/Prototypes/Entities/Objects/Specific/Medical/healthanalyzer.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
 - type: entity
   id: HandheldHealthAnalyzer
-  parent: [ HandheldHealthAnalyzerUnpowered, PowerCellSlotSmallItem]
```
Comment (Emisse) [inline Resources/Prototypes/Entities/Objects/Specific/Medical/healthanalyzer.yml:42]: keep this small

**Review by Emisse — CHANGES_REQUESTED** [review]

**Comment by SonicHDC** [comment]
Maybe better adding something new before removing what's exist?

**Comment by JoeHammad1844** [comment]
image
I feel we should listen to the audience here. and if you dont, add a health analyzer to every doctors backpacks

**Comment by Partmedia** [comment]
I think most of the feedback suggests that people would really prefer the PDA scanners to stay. For better or worse, the health analyzers are pretty essential to what MED does because of the lack of other obvious signs of problems. Maybe we can revisit this when there are better ways to diagnose patients.

**Comment by Partmedia** [comment]
Closing per the previous comment. Please re-open if more discussion is needed on this.

**Pull request closed without merging**